composer.json

```diff
@@ -18,16 +18,15 @@
         "publishing"
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.4 || ^8.0",
         "silverstripe/framework": "^4.6",
         "silverstripe/cms": "^4",
         "silverstripe/config": "^1",
         "symbiote/silverstripe-queuedjobs": "^4.5",
         "silverstripe/versioned": "^1.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
-        "squizlabs/php_codesniffer": "^3"
+        "silverstripe/recipe-testing": "^1 || ^2"
```

Comment:
```suggestion
        "silverstripe/recipe-testing": "^2"
```

Reply:
Though does this even need recipe-testing? Isn't `"phpunit/phpunit": "^9.5"` enough?

Reply:
Not sure. I just went with Mojmir's recommendation.